lib/Model/Constraint.php

```diff
@@ -371,22 +372,23 @@ public function setValue($value)
     /**
      * Returns true if offset exists. False otherwise.
      *
-     * @param integer $offset Offset
+     * @param int|string $offset Offset
      *
      * @return boolean
      */
-    public function offsetExists($offset)
+    public function offsetExists($offset): bool
     {
         return isset($this->container[$offset]);
     }
 
     /**
      * Gets offset.
      *
-     * @param integer $offset Offset
+     * @param int|string $offset Offset
      *
      * @return mixed
      */
+    #[\ReturnTypeWillChange]
```

Comment:
Had to add this for PHP 7.4 support since the `mixed` return type does not exist there.